source/lucidity/__init__.py

```diff
@@ -2,127 +2,31 @@
 # :copyright: Copyright (c) 2013 Martin Pengelly-Phillips
 # :license: See LICENSE.txt.
 
-import os
-import uuid
-import imp
-
 from ._version import __version__
-from .template import Template, Resolver
-from .error import ParseError, FormatError, NotFound
-
-
-def discover_templates(paths=None, recursive=True):
```

Comment:
I actually had these in `__init__.py` as the 'high level' API that operate on template instances rather than being templates themselves.

So I don't really see a benefit to changing this at this stage.